src/Sylius/Bundle/GridBundle/Renderer/TwigGridRenderer.php

```diff
@@ -92,7 +93,12 @@ public function render(GridView $gridView, $template = null)
      */
     public function renderField(GridView $gridView, Field $field, $data)
     {
-        return $this->fieldsRegistry->get($field->getType())->render($field, $data);
+        $fieldType = $this->fieldsRegistry->get($field->getType());
+        $resolver = new OptionsResolver();
+        $fieldType->configureOptions($resolver);
+        $options = $resolver->resolve($field->getOptions());
```

Comment:
I think eventually both the retrieval of the field type service and the resolution of the options could be done at a higher level, and so the method signature for the rednerer could be something like:

```php
public function renderField(GridView $gridView, Field $field, FieldTypeInterface $type, array $options, $data)
```

But nothing to do with this PR.